### DISCUSSION TITLE
Problem with St Lucia Spa

### DISCUSSION THREAD

**Discussion by Everdol** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#728] [Q&A]
Hello, i am having many problems to connect properly the PCB V2 kit with my st Lucia spa (6 wires). 
I can manage the web interface but when i install it the kit in the spa, doesn't works. The screen in the spa still off.
Thanks in advance for any help, and sorry for my poor english.
![IMG_20240612_170729833~2](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/170173706/2d427fce-d1b0-4d96-a2a4-b1013bf009a5)
![IMG_20240612_170406164~2](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/170173706/ef409cb3-e9ab-4e8c-8bcd-8e2f9b417f35)

**Comment by visualapproach** [comment]
Your English is fine. But you connections are not working. It may be bad solder joints. Can we see the back side of the PCB?

**Reply by visualapproach** [reply]
Actually, when looking closer- have you even soldered the pins to the level converter?

**Reply by Everdol** [reply]
Yes, I have soldered all the pins.
![17182075614291245996652040964975](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/170173706/cb9a9fec-9c1d-450e-8ae6-a3e896aba51a)

**Reply by visualapproach** [reply]
Thank you! Not to be rude, but the solder joints look really bad. Use flux and make sure both the pin and plates are wetting (solder hugging the metal rather than curling up into a ball).
And I'm curious about the LLC. It looks like you just shoved the pins through without soldering them to the red pcb. Or did you solder them on the backside of the red pcb?
